about/faq.md

```diff
@@ -23,134 +19,93 @@ platform, as well as different fixes for its supported devices. Quoting the
+In 2017, engineers from [Igalia](https://igalia.com) submitted a new flavor of WPE, suitable for upstream hosting under the [webkit.org](https://webkit.org/) umbrella. This version of WPE is [released every 6 months](/release/schedule) from the code hosted on the [upstream repository](https://webkit.org). This flavor of WPE is maintained upstream and provides regular [security updates](/security).
+
 
 ## Is WPE supported on any specific hardware System-on-Chip?
 
-WPE has been [ported to a wide range of hardware
-platforms](https://wpewebkit.org/about/supported-hardware.html). The team aims
-to expand the list even further, don't hesitate to contact us if you can't find
-your favorite SoC in the list.
+WPE has been [ported to a wide range of hardware platforms](https://wpewebkit.org/about/supported-hardware.html). The team aims to expand the list even further, so don't hesitate to contact us if you can't find your favorite <acronym title="System-on-Chip">SoC</acronym> in the list.
+
 
 ## Is WPE ported to non-Linux operating systems?
 
-WPE currently only works on Linux-based operating systems. We are currently
-working on supporting Android though. If you require WPE to run on any other OS,
-don't hesitate to contact us.
+WPE currently only works on Linux-based operating systems. We are currently working on supporting Android, though. If you require WPE to run on any other OS, don't hesitate to contact us.
+
 
 ## What Web features does WPE support?
 
-The WebKit project provides a [features status page](https://webkit.org/status/)
-to which you can refer. However, these informations are to take with a grain of
-salt. The multi-platform nature of WebKit implies that ports have different
-build-time and runtime configurations.
+The WebKit project provides a [features status page](https://webkit.org/status/) to which you can refer. However, that information should be taken with a grain of salt. The multi-platform nature of WebKit implies that ports have different build-time and runtime configurations.
+
+The WPE project currently does not have an official list of the Web features it supports. It might provide one in the future, but for the time being, we recommend users check for specific features by testing WPE through the Cog browser.
 
-The WPE project currently does not have an official list of the Web features it
-supports. We might provide one, but for the time being we recommend users to
-check for specific features themselves by testing WPE through the Cog browser.
 
 ## What's the status regarding WebRTC? 
 
 As of end of 2020 the facts are:
 
-1. [WPEWebKit upstream has support for WebRTC](https://blogs.gnome.org/tsaunier/2018/07/31/webkitgtk-and-wpe-gains-webrtc-support-back/), by relying on LibWebRTC.
+1. [WPEWebKit upstream has support for WebRTC](https://blogs.gnome.org/tsaunier/2018/07/31/webkitgtk-and-wpe-gains-webrtc-support-back/), by relying on [LibWebRTC](https://github.com/aisouard/libwebrtc).
```

Comment:
The upstream libwebrtc code is hosted on https://webrtc.googlesource.com/src/

Reply:
Link updated and pushed.